plat/nvidia/tegra/include/platform_def.h

```diff
@@ -47,11 +47,11 @@
 
 #define TEGRA_PRIMARY_CPU		0x0
 
-#define PLATFORM_MAX_AFFLVL		MPIDR_AFFLVL2
-#define PLATFORM_CORE_COUNT		PLATFORM_MAX_CPUS_PER_CLUSTER
-#define PLATFORM_NUM_AFFS		((PLATFORM_CLUSTER_COUNT * \
-					  PLATFORM_CORE_COUNT) + \
-					  PLATFORM_CLUSTER_COUNT + 1)
+#define PLATFORM_MAX_AFFLVL		MPIDR_AFFLVL1
+#define PLATFORM_CORE_COUNT		(PLATFORM_CLUSTER_COUNT * \
+					 PLATFORM_MAX_CPUS_PER_CLUSTER)
+#define PLATFORM_NUM_AFFS		(PLATFORM_CORE_COUNT + \
+					 PLATFORM_CLUSTER_COUNT)
```

Comment:
I've just been reminded that the Tegra port supports PSCI SYSTEM_SUSPEND:
https://github.com/ARM-software/arm-trusted-firmware/blob/master/plat/nvidia/tegra/common/tegra_pm.c#L122

So I guess `PLATFORM_MAX_AFFLVL` should be `MPIDR_AFFLVL2` as it currently is. Perhaps the correct fix is as follows:
```
#define PLATFORM_MAX_AFFLVL	MPIDR_AFFLVL2
#define PLATFORM_CORE_COUNT	(PLATFORM_CLUSTER_COUNT * \
								PLATFORM_MAX_CPUS_PER_CLUSTER)
#define PLATFORM_NUM_AFFS		(PLATFORM_CORE_COUNT + \
								PLATFORM_CLUSTER_COUNT + 1)
```
Could you test SYSTEM_SUSPEND and confirm?

Thanks

Dan.